docs/services/frame-system/_index.md

```diff
@@ -86,13 +86,6 @@ The `Orientation` parameter offers `Types` for ease of configuration, but the Fr
 
 {{% /alert %}}
 
-{{% alert title="Tip" color="tip" %}}
-
-Viam's coordinate system considers `+X` to be forward, `+Y` to be left, and `+Z` to be up.
-You can use [the right-hand rule](https://en.wikipedia.org/wiki/Right-hand_rule) to determine the orientation of these axes.
-
-{{% /alert %}}
-
```

Comment:
I think this still need to be defined somewhere. Removing the definition just makes it MORE ambiguous. Frames themselves are entirely neutral, but we should have a standard for how to attach them to physical objects so that they line up consistent with things humans would consider "forward" or "left" or "positive spin". This was the real issue in my original complaint. We have to say (somewhere, or even multiple places for different things) HOW a frame attaches to a physical object with human-like assumptions about intersectionality, otherwise our components are no longer interchangeable.

tl;dr I believe it should be absolutely clear, to a viam novice reading docs alone, how orientation/direction maps to a component and human assumptions (even if that varies per type/model.)

Reply:
Hmm. The note as-is is wrong though since Viam has no such consistent coordinate system as far as I understand. As you noted, bases have y forward, but then I think we have grippers with x "forward" (depending on what you even mean by forward :D )
Whereas with differential drive bases that can only move efficiently along one axis, arms and grippers and things have no such inherent axis.

Reply:
I think we can potentially solve that by adding specificity to the alert and explaining how it works for different components. I think @Otterverse's point is valid that this does help understanding. Can you work out how it works for each involved resource and potentially also check with the SME's whether we can make it consistent if it is not?

Reply:
What I did in this PR was based on conversations yesterday with Peter. For bases, Y is forwards, but for everything else, there is no inherent forwards. Should we say that?